spark-cassandra-connector/src/test/scala/com/datastax/spark/connector/rdd/partitioner/TokenRangeClustererTest.scala

```diff
@@ -52,50 +54,38 @@ class TokenRangeClustererTest {
 
   @Test
   def testSplitByCount() {
-    val tr1 = new TokenRange(token(0), token(10), Set(node1), 5)
-    val tr2 = new TokenRange(token(10), token(20), Set(node1), 5)
-    val tr3 = new TokenRange(token(20), token(30), Set(node1), 5)
-    val tr4 = new TokenRange(token(30), token(40), Set(node1), 5)
+    val tr1 = tokenRange(start = minToken, end = minToken / 2, Set(node1))
+    val tr2 = tokenRange(start = minToken / 2, end = 0, Set(node1))
+    val tr3 = tokenRange(start = 0, end = maxToken / 2, Set(node1))
+    val tr4 = tokenRange(start = maxToken / 2, end = maxToken, Set(node1))
 
-    val trc = new TokenRangeClusterer[Long, LongToken](10)
+    val trc = new TokenRangeClusterer[Long, LongToken](2)
     val groups = trc.group(Seq(tr1, tr2, tr3, tr4)).map(_.toSet).toSet
     assertEquals(2, groups.size)
     assertTrue(groups.contains(Set(tr1, tr2)))
     assertTrue(groups.contains(Set(tr3, tr4)))
   }
 
   @Test
-  def testTooLargeRanges() {
```

Comment:
As `TokenRange` no longer carries `dataSize` I couldn't find a reasonable corresponding alternative to this test.